src/gatsby/src/templates/park.js

```diff
@@ -163,10 +163,9 @@ export default function ParkTemplate({ data }) {
   }, [apiBaseUrl, park.orcs])
 
   useEffect(() => {
-    if (window.location.hash && !isLoadingProtectedArea && !isLoadingAdvisories) {
-      const hashSave = window.location.hash
-      window.location.hash = ""
-      window.location.hash = hashSave
+    if (window.location.href && !isLoadingProtectedArea && !isLoadingAdvisories) {
```

Comment:
`window.location.href` will always be true.  Are you sure the if condition needs to change?

Reply:
Good catch, it works without it.